nixos/modules/programs/openlinkhub.nix

```diff
@@ -0,0 +1,52 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+let
+  cfg = config.programs.openlinkhub;
+  pkg = config.programs.openlinkhub.package;
+in
+{
+  options.programs.openlinkhub = {
+    enable = lib.mkEnableOption "openlinkhub";
+    package = lib.mkPackageOption pkgs [ "delta" "openlinkhub" ] { };
```

Comment:
There seems to be a bunch of configurable stuff at https://github.com/jurkovic-nikola/OpenLinkHub/blob/cee9c84553021f014bd3465eb3a2b2be75a52b03/src/config/config.go#L10-L40. Could you add a freeform json `settings` option?